driver/examples/configs/analytic_test.yaml

```diff
@@ -0,0 +1,101 @@
+stencil_config:
+  compilation_config:
+    backend: numpy
+    rebuild: false
+    validate_args: true
+    format_source: false
+    device_sync: false
+initialization:
+  type: analytic
+  config:
+    case: baroclinic
+performance_config:
+  collect_performance: true
+  experiment_name: c12_baroclinic
+comm_config:
+  type: null_comm
+  config:
+    rank: 0
+    total_ranks: 6
+nx_tile: 12
+nz: 79
+dt_atmos: 225
+minutes: 15
+layout:
+  - 1
+  - 1
+diagnostics_config:
+  path: output
+  output_format: netcdf
+  names:
+    - u
+    - v
+    - ua
+    - va
+    - pt
+    - delp
+    - qvapor
+    - qliquid
+    - qice
+    - qrain
+    - qsnow
+    - qgraupel
+  z_select:
+    - level: 65
+      names:
+        - pt
```

Comment:
The total number of diagnostic quantities available within the native version is staggering.  Might want to consider making the diagnostics_config entry eventually live in its own yaml file.

Reply:
Raised this as an issue for after this PR